ADRs/RH/SIG-SRE/ADR-00005 SLO Lifecycle.md

```diff
@@ -0,0 +1,153 @@
+
+* Sometimes the cause might be a result of an upstream quality or reliability issue that needs to be escalated to the team or vendor responsible.
+* Sometimes it may be a result of a defect that was deployed.
+* Sometimes it may be a result of insufficient capacity to meet the needs of the userbase that can be resolved by a capacity planning cycle.
+
+Whatever the cause, the [RACI chart][raci-chart] can help inform who the responsible parties are for remediation.
+
+#### Sprint Planning Hooks
+
+A primary goal of Service Level Objectives is to inform the natural cycle of work for product feature teams by adding an additional signal for use in planning work. The new signal is the service's remaining Error Budget.
+
+The "elevator pitch" flow chart looks like this: If there's error budget remaining, it's a green light to focus on features, otherwise, focus on reliability.
+
+```mermaid
+graph TD
+  A(Sprint Planning) --> B{Error Budget<br />Available?}
+  B --> |Yes|C[Features]
+  B --> |No|D[Reliability]
+  C & D --> E(Prioritise the appropriate work)
+```
+
+In reality, however, the error budget is not meant to be the _only_ signal feeding sprint planning; it is merely _another_ signal that can be used to prioritise work. Planning is still up to the team and can focus on features when it makes sense. Sprint planning may look like this with the error budget informing decisions.
+
+```mermaid
+  graph TD
+  A(Sprint planning) ---> B[[Review product<br />roadmap needs]]
+  C[/Remaining error<br />budget/] -.->  |Informs|D[[Backlog grooming]] & B
+  B --> D
+  D --> E[[Pull cards<br />into sprint]]
+  E --> F(Start sprint)
+```
+
+A parting thought for this section is to remember the purpose of the Service Level Objective and associated error budget is to model customer satisfaction. Use caution when deprioritizing reliability for features when the error budget is depleted, or projected to become exhausted. Those who are closest to the service will have the best understanding of any impact for delaying reliability enhancements.
+
+## Challenges
+
+* Teams may desire to adopt the guidelines in this document but lack the data to do so because they don't have any SLO, or a way to measure the remaining error budget. 
+* Teams may face encounter difficulties scheduling reliability work by those who deprioritize it in favour of "feature work."
+  * Related, teams may not have the throughput to address reliability work above "keeping the lights on" work. (See also toil management.)
+
+These challenges could be addressed by a [permissive SLO phase](#permissive-slo-phase).
+
+### Permissive SLO Phase
+
+Teams may find that adopting this SLO lifecycle challenging for any number of reasons, whether they're listed in the [challenges](#challenges) section or not. For teams that want adopt a lightweight approach to the SLO lifecycle, a "permissive phase" could be helpful.
```

Comment:
Yeap, just as a concept though, they can be implemented to help calibrate your actual SLOs better

Reply:
@dastergon I've had a think and maybe we leave that out of the lifecycle doc here and give the idea a home in the kind of slo cookbook, since I know we have teams that chose that path.

Reply:
Let's table the aspirational SLOs for the moment only because we can add it later to either this, or the cookbook once we have a worked example.